zfit_physics/models/pdf_cmsshape.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+import numpy as np
+import tensorflow as tf
```

Comment:
Hm, would be good to have znp. but erf and erfc isn't directly available in the namespace, we should add this in zfit. But TF is fine, it's straight forward to convert

Reply:
Unless you want to add more stuff into the `znp` namespace we keep it that way for now.